### DISCUSSION TITLE
Where are "suspend file watching" and "suspend database reflecting"

### DISCUSSION THREAD

**Discussion by gerroon** [vrtmrz/obsidian-livesync#329] [Documentation requests] [answered]
Hi

When I open Obsidian on IOS i get a message about these being turned on but I can't find these settings in the plugin pages. I am guessing that the actual settings and the names do not match.

**Comment by dlphnkck** [comment, marked as answer]
[![Image from Gyazo](https://i.gyazo.com/842c27c4ad3ca33e7534735de8251a3b.png)](https://gyazo.com/842c27c4ad3ca33e7534735de8251a3b)

1. Settings for Self-hosted LiveSync→"Hatch"tab
	- Suspend file watching 
	- Suspend database reflecting

2. If you had performed a search before, you would have found clues like "Suspend file watching" or "Suspend database reflecting" below.

[Sync only works in 1 direction · Issue \#241 · vrtmrz/obsidian\-livesync](https://github.com/vrtmrz/obsidian-livesync/issues/241#issuecomment-1635130081)
> please make sure `Suspend file watching` have been disabled just in case. It is on the `Hatch` pane in the setting dialogue.

https://github.com/vrtmrz/obsidian-livesync/blob/ac9428e96b94c92f8f7a7dabdb3437a073e4016c/src/ObsidianLiveSyncSettingTab.ts
Under the "Hatch" tab, you can locate the options to "Suspend file watching" and "Suspend database reflecting."

3. There appears to be a plugin designed to search for items in the settings. I have not utilized it, but it may potentially be beneficial to you.

[javalent/settings\-search: Adds a search bar to Obsidian\.md's settings](https://github.com/javalent/settings-search)
> The Settings Search plugin adds global search functionality to the Obsidian settings, allowing users to quickly search for a particular setting.

**Reply by gerroon** [reply]
Thanks, I appreciate the detailed answer. I kept looking at in the sync settigns
